Fixes #39 Scene lighting needs improvement

Yahiewi opened the pull request:
This PR adds a Lights folder to give the user the option to manually control the lighting parameters for the three different light sources (Directional, Ambient and Hemispherical) as well as to visualize them using helpers.
For this, there was a need to modify the base CSS class to accommodate the new folder and add make the control panel scrollable and its close button visible.
This way, the user can adjust the lighting settings to their needs depending on the model being viewed.
I also tweaked the initial settings for the lighting to give the models a more "realistic" feel with better contrast and shadows.
For models that are completely white (like the Panda model), changing the background color of the scene (as lighting will have a much less important effect here) is a good solution that improves the contrast and visibility of the object.

IsabelParedes commented on src/controls.ts:
```diff
@@ -207,4 +219,149 @@ export class URDFControls extends GUI {
+            angleStep
+          )
+        },
+        target: {
```
99% of the time, the robot will be at the origin, so we can probably remove these $x$, $y$, $z$ options

Yahiewi replied:
I removed the target position controls so the directional lights always aim at the origin.

IsabelParedes commented on src/controls.ts:
```diff
@@ -207,4 +219,149 @@ export class URDFControls extends GUI {
+
+      // Hemisphere light controls
+      this.controls.lights.hemisphere = {
+        skyColor: hemisphereFolder.addColor(hemisphereSettings, 'SkyColor'),
```
For the text displayed in the GUI, make the labels separate words
```
Sky Color
```

Yahiewi replied:
Added a .name("Text") to solve this.

IsabelParedes commented on src/controls.ts:
```diff
@@ -51,6 +53,9 @@ export class URDFControls extends GUI {
     this._jointsFolder = this.addFolder('Joints');
     this._jointsFolder.domElement.setAttribute('class', 'dg joints-folder');
+
+    this._lightsFolder = this.addFolder('Lights');
```
Since there exists a `Scene` folder already that affects some of the lights, it makes more sense to add the new folders to `Scene` instead of creating a new `Lights` folder.

IsabelParedes replied:
Also, since `Joints` is the main priority and now that the gui is getting longer, the folders should be reorganized in this order.
1. Joints
2. Workspace
3. Scene

Yahiewi replied:
Did as you suggested.

Yahiewi commented:
Addressed all the suggestions and fixed a major CSS incompatibility caused by this PR (specifically when I made the controls panel scrollable) and the PR that makes the controls panel resizeable. Couldn't solve this through CSS, so solved it in the _setupResizeHandling function in controls.ts by writing the logic for the cursor there.

IsabelParedes commented:
Resolves #39
